tests/Functional/ImportCommandsTest.php

```diff
@@ -25,12 +25,7 @@ public function testImportDatabase()
     {
         $backupUrl = $this->getBackupUrl('database');
 
-        $importDatabaseCommand = sprintf(
-            'import:database %s.%s "%s"',
-            $this->getSiteName(),
-            'live',
-            $backupUrl
-        );
+        $importDatabaseCommand = sprintf('import:database %s "%s"', $this->getSiteEnv(), $backupUrl);
```

Comment:
for what it's worth, these multi-line function calls were most likely on multiple lines to comply with "lines should be less than 80 characters" style guides..

Reply:
The linting (phpcs) ruleset is configured to `120` limit (`tests/config/linting_ruleset.xml`)